backend/collab_service/server.js

```diff
@@ -12,6 +12,13 @@ const io = socketIo(server, {
     }
   });
 
+const collabRoutes = require('./routes/collabRoutes');
```

Comment:
It would be ideal if you could shift this file's logic to app.js instead, so that it follows the style we adopted in other services as well. 😄

Reply:
have refactored the collab svc! tqtq